tests/0004-conf.c

```diff
@@ -381,6 +381,25 @@ int main_0004_conf (int argc, char **argv) {
 		rd_kafka_conf_destroy(conf);
 	}
 
+	{
+		rd_kafka_conf_res_t res;
+
+		TEST_SAY("Error reporting for S2F properties\n");
+		conf = rd_kafka_conf_new();
+
+		res = rd_kafka_conf_set(conf, "debug", "invalid-value", errstr, sizeof(errstr));
```

Comment:
maybe strap some valid flags in there as well:
"cgrp,invalid-value,topic"

and then add another assertation:
`TEST_ASSERT(!strstr(errstr, "cgrp") && !strstr(errstr, "topic"))

Reply:
Good idea!